src/main/java/hmim/eteam/rest/backend/api/RoleResolver.java

```diff
@@ -12,7 +13,7 @@
 import javax.validation.constraints.NotNull;
 import java.util.Optional;
 
-public class RoleResolver {
+class RoleResolver implements IRoleResolver {
     private final AuthTokenRepository authTokenRepository;
     private final CourseRepository courseRepository;
     private final CourseRoleRepository courseRoleRepository;
```

Comment:
From my experience, abbreviation usage is forbidden in lots of codestyles (not only in java), because in large scale it leads to "strange dirty naming".